modules/platform/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/BatchUploadFixture.java

```diff
@@ -1053,6 +1023,21 @@ public void testErrorOnRefreshedTokenError() throws Exception {
         }
     }
 
+    @Test
+    /** NXP-29246: Fix import of MHTML file using Chrome */
```

Comment:
Javadoc always above the annotations.

Note sure the JIRA title is needed as the test method explicitly state that it tests a MHTLM upload.